src/TodoList.tsx

```diff
@@ -43,11 +48,16 @@ export const TodoList: React.FC = () => {
           </li>
         ))}
       </ul>
-      <input
+      <Input
+        id={Date.now()}
+        placeholder="Enter Todo Item"
+        onChange={handleInputChange}
+      />
+      {/* <input
         type="text"
```

Comment:
We need a prop to represent the `type` parameter on a standard HTML `<input>`. This is the prop that determines if the input is text, date, etc. Check out the documentation here: [input docs](https://www.w3schools.com/tags/tag_input.asp)